mod/metrics/set/type_plus_right/metric/formula/wiki_rating.rb

```diff
@@ -18,7 +18,18 @@
   # table with Metrics on left and Weight inputs on right
   def rating_editor_table
     table rating_editor_table_content, class: "wikiRating-editor",
-                                       header: %w[Metric Weight]
+                                       header: ["Metric", checkbox_equalizer]
+  end
+
+  def checkbox_equalizer
+    "<span>Weight</span>
+    <div class='form-check checkbox-pull pull-right' >
+      <label class='form-check-label'>
+        <input type='checkbox' id='equalizer' checked >
+        <span style='font-weight: bold;' >Equalize</span>
+      </label>
+    </div>
+    "
```

Comment:
we're trying not to write this much HTML in Ruby.  Other options include:
1. use Rails API (eg `check_box_tag`)
2. use HAML templates.
3. If you must use HTML follow the `<<-HTML` pattern.